client/blocks/eligibility-warnings/hold-list.tsx

```diff
@@ -151,16 +235,57 @@ export const HoldList = ( { holds, isPlaceholder, translate }: Props ) => {
 							</div>
 						)
 					) }
-			</Card>
-		</div>
+			</div>
+		</>
 	);
 };
 
+function getCardHeading( context: string | null, translate: LocalizeProps[ 'translate' ] ) {
+	const defaultCopy = translate( 'Please clear all issues above to proceed.' );
```

Comment:
Unfortunately, this copy (`Please clear all issues above to proceed.`) doesn't work since the issues now appear below the Heading text.  I'll just update it to the new fallback: `To continue you'll need to:` & hopefully the individual items will be enough context for non-English speakers.

Reply:
This should be fixed in 3a756ef